docs/manual-bridging.md

```diff
@@ -75,21 +83,56 @@ Note: The bridge only supports standard ERC-20 tokens, so make sure your token a
         - Bridge fee estimate in ETH (see [Estimating Bridge Fee](#estimating-bridge-fee) for details)
     - `amount`: Amount to transfer in wei e.g. 1 ETH would be represented as `100000000000000`
     - **Note:** The `depositETH()` function will transfer funds to the same address on L2 as the sender executing this transaction on L1. If however, you’d like to deposit to a different address use the `depositToETH`function, which will take an additional parameter `receiver` which is the address of the receiver on L2.
-- The transaction will first transfer the specified amount of ERC-20 from the user to the bridge and perform a cross-chain call to the L2 to mint a corresponding amount of tokens to the intended receiver address.
```

Comment:
Are we deleting this mostly just cuz it's kind of unnecessary information?

Reply:
Yeah, in the case of ERC-20, I think this information adds some value (earlier section) as it implicitly explains why an approve is required as a pre-step. Here however, I don't think it does.